woocommerce-pdf-invoices-packingslips.php

```diff
@@ -384,6 +385,56 @@ public function nginx_detected()
 		}
 	}
 
+	/**
+	 * Detect MailPoet.
+	 * @return void
+	 */
+	public function mailpoet_mta_detected() {
+		$mailpoet = $this->namespace_exists( 'MailPoet' );
```

Comment:
What's the reason you're checking the namespace? I think you can directly move to the `is_callable` check:
```php
if( is_callable( array( '\\MailPoet\\Settings\\SettingsController', 'getInstance' ) ) ) {
```
with that you don't need `class_exists` either.

Reply:
Fixed and tested.